src/hrc721.spec.ts

```diff
@@ -71,4 +74,48 @@ describe('HarmonyProvider', () => {
       expect(provider.ownerOf('')).to.be.rejectedWith(Error)
     })
   })
+
+  describe('transferFrom', () => {
+    it('should throw if there is no signer', () => {
+      expect(provider.transferFrom(TEST_ADDRESS_1, RESULT_TEST_ADDRESS, '1')).to.be.rejectedWith(Error)
+    })
+
+    it('should transfer the ownership of a token from one address to another', async () => {
+      const owner = await provider.ownerOf('5')
+      expect(owner).to.be.oneOf([TEST_ACCOUNT_2.address, TEST_ACCOUNT_3.address])
+
+      const ownerAccount = [TEST_ACCOUNT_2, TEST_ACCOUNT_3].find((account) => account.address === owner)
+      const receiverAccount = [TEST_ACCOUNT_2, TEST_ACCOUNT_3].find((account) => account.address !== owner)
+      if (!ownerAccount || !receiverAccount) throw new Error('Account not found')
+
+      provider.setSignerByPrivateKey(ownerAccount.privateKey)
+      const result = await provider.transferFrom(ownerAccount.address, receiverAccount.address, '5')
+
+      expect(result.txStatus).to.eq(TxStatus.CONFIRMED)
+      expect(result.receipt).to.exist
+      expect(result.receipt?.blockHash).to.be.string
+    }).timeout(30000)
```

Comment:
this is not need it. Add a greater timeout in the mocha command like [this](https://github.com/blockcoders/near-rpc-providers/blob/main/package.json#L17-L18).